pkg/virt-controller/watch/vm/vm.go

```diff
@@ -3255,6 +3263,12 @@ func (c *Controller) sync(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineI
 		vm = vmCopy
 	}
 
+	if vmi == nil {
+		if downwardmetrics.IsDownwardMetricsConfigurationInvalid(c.clusterConfig, &vm.Spec.Template.Spec) {
+			return vm, common.NewSyncError(fmt.Errorf("DownwardMetrics feature is not enabled"), controller.FeatureNotEnabled), nil
```

Comment:
`fmt.Errorf("DownwardMetrics feature is not enabled")`

Make this a const or a custom error in the downwardmetrics pkg?

Reply:
Done. I've also replaced all occurrences of this string with the custom error variable.